include/irr/asset/IDescriptorSetLayout.h

```diff
@@ -0,0 +1,45 @@
+#ifndef __IRR_I_DESCRIPTOR_SET_LAYOUT_H_INCLUDED__
+#define __IRR_I_DESCRIPTOR_SET_LAYOUT_H_INCLUDED__
+
+#include "irr/asset/ShaderCommons.h"
+#include "irr/asset/SSamplerParams.h"
+
+namespace irr { namespace asset
+{
+
+enum E_DESCRIPTOR_TYPE
+{
+    EDT_SAMPLER = 0,
```

Comment:
ok we cant support separable sample from image

Reply:
@Crisspl TODO

Reply:
resolved